tests/acceptance.suite.dist.yml

```diff
@@ -37,5 +37,5 @@ modules:
             admin email: 'admin@mydomain.com'      # email Id of the Admin
             language: 'English (United Kingdom)'   # Language in which you want the Application to be Installed
         AcceptanceHelper:
-            repo_folder: '/home/travis/build/redCOMPONENT-COM/redCORE/' # Path to the Extension to be installed via Install from folder
+            repo_folder: '/home/travis/build/redCOMPONENT-COM/redCORE/.dist/redCORE' # Path to the Extension to be installed via Install from folder
```

Comment:
If you want to install the .zip

![screen shot 2015-05-20 at 12 30 35](https://cloud.githubusercontent.com/assets/1375475/7723744/31756cf8-feeb-11e4-9188-18f299ad92ec.png)

This needs to be modified: https://github.com/redCOMPONENT-COM/redCORE/blob/develop/tests/acceptance/02-InstallExtensionCept.php#L14-L15

Right now it fails in local, because is still trying to install from folder:

![screen shot 2015-05-20 at 12 29 50](https://cloud.githubusercontent.com/assets/1375475/7723764/5c8ea10c-feeb-11e4-8457-4f7f3f168d8b.png)

Not sure how it passed in Travis, in local fails:

![screen shot 2015-05-20 at 12 33 07](https://cloud.githubusercontent.com/assets/1375475/7723784/8f128cce-feeb-11e4-888b-c803a263f56f.png)